views/random-beer.hbs

```diff
@@ -0,0 +1,12 @@
+{{#each responseFromAPI}}
+
+{{> beerpartial }}
+
+<p>
+    {{food_pairing}}
+</p>
+
+<p>
+    {{brewers_tips}}
+</p>
+{{/each}}
```

Comment:
A small thing, but it would be nice to have the info introduced, as opposed to just pasted in there. So for instance: <p> Brewer's tips: {{ brewers_tips}} </p>